docs/user/rstudio/ide/guide/code/console.qmd

```diff
@@ -2,43 +2,39 @@
 title: "R Console"
 ---
 
-## Overview
-
-The Console in the RStudio includes a variety of features intended to make working with R more productive and straightforward. This article reviews these features. Learning to use these features along with the related features available in the [Source](https://support.rstudio.com/hc/en-us/articles/200484448) and [History](https://support.rstudio.com/hc/en-us/articles/200526217) panes can have a substantial payoff in your overall productivity with R.
-
-Copied from: <https://support.rstudio.com/hc/en-us/articles/200404846-Working-in-the-Console>
+The **Console** in RStudio includes a variety of features intended to make interactively working with code more productive and straightforward. This documentation reviews these features. Learning to use these features along with the related features available in the [Source](execution.html) pane can have a substantial payoff in your overall productivity.
 
 ## Code Completion
 
-The RStudio supports the automatic completion of code using the **Tab** key. For example, if you have an object named `pollResults` in your workspace you can type `poll` and then **Tab** and RStudio will automatically complete the full name of the object.
+RStudio supports the automatic completion of code using the **Tab** key. For example, if an object named `pollResults` exists in the workspace, typing `poll` in the R console and then pressing **Tab** will automatically complete the full name of the object.
```

Comment:
`Tab` for autocompletion? 🤯

I thought only `Enter` worked!

Reply:
So many goodies across all the support articles!!